RecordPoint.Connectors.SDK.Client.Abstractions/Providers/ISdkAzureBlobRetryProvider.cs

```diff
@@ -1,11 +1,8 @@
-﻿using Microsoft.Azure.Storage.Blob;
-
-namespace RecordPoint.Connectors.SDK.Providers
+﻿namespace RecordPoint.Connectors.SDK.Providers
```

Comment:
Note: May need to confirm that the namespace change doesn't break any references.